OpenSim/Sandbox/MatlabScripts/opensimTableToArray.m

```diff
@@ -0,0 +1,35 @@
+function [data_array,labels] = opensimTableToArray(table)
+
+
+
+
+
+import org.opensim.modeling.*
+
+% allocate an array
+nCol = table.getNumColumns;
+nRow = table.getNumRows;
+data = zeros(nRow, nCol);
+labels = {};
+
+for iCol = 0 : nCol - 1
+
+    for iRow = 0 : nRow - 1
+
+        data(iRow+1,iCol+1) = table.getDependentColumnAtIndex(iCol).getElt(iRow,1);
```

Comment:
Thanks for having a look. If an OBE is made, Matlab has a catastrophic failure. The whole crashes without a discernable message. This makes it tough to debug if you have a big program with lots for loops. 

![image](https://cloud.githubusercontent.com/assets/5296507/18459096/8dec60f8-791b-11e6-8e14-a74524be08f6.png)